bin/p2-bootstrap/bootstrap.go

```diff
@@ -87,7 +89,7 @@ func main() {
 
 func InstallConsul(consulPod *pods.Pod, consulManifest pods.Manifest) error {
 	// Inject servicebuilder?
-	err := consulPod.Install(consulManifest)
+	err := consulPod.Install(consulManifest, auth.NoopVerifier())
```

Comment:
super duper nit but the `ioutil` package uses `Nop` instead of `Noop`: https://golang.org/pkg/io/ioutil/#NopCloser